src/Nethermind/Nethermind.Network.Enr/NodeRecord.cs

```diff
@@ -27,7 +27,7 @@ namespace Nethermind.Network.Enr;
 /// </summary>
 public class NodeRecord
 {
-    private int _enrSequence;
+    private long _enrSequence;
```

Comment:
there are comments for GetValue and GetObj, I think -> they cannot be implemented with the same method because one is returning a nullable and the other a nullable reference

Reply:
Moved the mentioned code to StringBuilder